core/common/src/LocalDate.kt

```diff
@@ -86,7 +86,15 @@ public expect class LocalDate : Comparable<LocalDate> {
         /**
          * Returns a [LocalDate] that is [epochDays] number of days from the epoch day `1970-01-01`.
          *
-         * @throws IllegalArgumentException if the result exceeds the platform-specific boundaries of [LocalDate].
+         * @throws IllegalArgumentException if the result exceeds the boundaries of [LocalDate].
```

Comment:
Do I get it correctly that currently the boundaries of `LocalDate` are not documented, but it is guaranteed to support all dates within `Instant.DISTANT_PAST` and `Instant.DISTANT_FUTURE`?

What about the `Instant`?

Reply:
The same goes for `Instant`: its boundaries are not documented, but `Instant.DISTANT_PAST` and `Instant.DISTANT_FUTURE` are guaraneteed to behave well.

The reason is that the only use cases for `Instant.MAX` and `Instant.MIN` we've encountered so far were:
* A sentinel "fake" value, in particular when looking for the maximum of the minimum value. `null` typically serves this purpose better.
* A shared concept of "a time so far away into the past/future that real data will certainly not be outside this boundary". `DISTANT_PAST` and `DISTANT_FUTURE` are ok for that.

Reply:
Thanks for the explanation!